src/compiler/crystal/command/playground.cr

```diff
@@ -34,6 +34,21 @@ class Crystal::Command
       end
     end
 
+    playground_url = "http://#{server.host || "localhost"}:#{server.port || "8080"}"
+
+    open_browser(playground_url)
+
     server.start
   end
+
+  # Open up the default browser automatically to the Playground.
+  private def open_browser(playground_url)
+    {% if flag?(:darwin) %}
+      system "open #{playground_url}"
+    {% elsif flag?(:win32) %}
+      system "start #{playground_url}"
+    {% elsif flag?(:linux) %}
+      system "xdg-open #{playground_url}"
```

Comment:
When this command fails (it does for me on Linux), it will print an ugly `--: 1: --: xdg-open: not found`.
This can be avoided by running the command without configuring it with an input, output or error. `system` configures it with an input, output or error as you can see [here](https://github.com/crystal-lang/crystal/blob/60760a5460aa13a1f80c5a1a7410782dc4076b77/src/process.cr#L536) (it uses [`Process::Redirect::Inherit`](https://crystal-lang.org/api/0.27.2/Process/Redirect.html#Inherit)) but `Process.run` doesn't do that by default and discards the IO (it uses [`Process::Redirect::Close`](https://crystal-lang.org/api/0.27.2/Process/Redirect.html#Close)).
```suggestion
      Process.run("xdg-open #{playground_url}", shell: true)
```